test/integration/react-streaming-and-server-components/switchable-runtime/pages/api/hello.js

```diff
@@ -0,0 +1,7 @@
+export default (req) => {
+  return new Response(`Hello from ${req.url}`)
```

Comment:
With the runtime set to `edge`, it seems the request `body` is always empty (e.g. `POST` requests), or am I missing something?